topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -349,35 +358,27 @@ you want to include Transcriptions factors in ChIP-seq experiments.
 > 3. Rename your dataset to reflect your findings
 {: .hands_on}
 
-You might have noticed that the UCSC file is in `BED` format and has a database associated to it. That's what we want for our peak file as well
```

Comment:
We do this to demonstrate the conversion feature - in respect to the "change filetype". As a bonus, the presenter can also demonstrate the implicit-conversion if wished.

Reply:
Ok I've added it back in. But imho if it's not necessary for users to do that conversion here  then it's just adding confusion and I'd demonstrate the conversion somewhere else where it is necessary.

Reply:
I disagree here. The `Get Flanks` tool is also not needed as a similar result can be archived by UCSC directly. It's important that people know the interface and this is the purpose of this tutorial, it's an introduction to Galaxy. 
Also please note that they are different intersect tools in Galaxy and not all can cope with interval files and hence convert them implicitly. It's therefore good to know that there are implicit and explicit conversions, maybe this should be made more clear in the text - I stress this a lot during my session.

Thanks for adding it back.

Reply:
I like the text. Thanks.